nipype/algorithms/stats.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Managing statistical maps
+"""
+from __future__ import (print_function, division, unicode_literals,
+                        absolute_import)
+import os
+import nibabel as nb
+import numpy as np
+
+from ..interfaces.base import (
+    BaseInterfaceInputSpec, TraitedSpec, SimpleInterface,
+    traits, InputMultiPath, File
+)
+
+
+class ACMInputSpec(BaseInterfaceInputSpec):
+    in_files = InputMultiPath(File(exists=True), mandatory=True,
+                              desc='input file, generally a list of z-stat maps')
+    threshold = traits.Float(1.65, usedefault=True,
+                             desc='binarization threshold. A z-value of 1.65 '
+                                  'corresponds to a two-sided test of p<.10')
+
+
+class ACMOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='output activation count map')
+    acm_pos = File(exists=True, desc='positive activation count map')
+    acm_neg = File(exists=True, desc='negative activation count map')
+
+
+class ACM(SimpleInterface):
+    """
+    Calculate a simple Activation Count Maps
+
+    Adapted from: https://github.com/poldracklab/CNP_task_analysis/\
+    blob/61c27f5992db9d8800884f8ffceb73e6957db8af/CNP_2nd_level_ACM.py
+    """
+    input_spec = ACMInputSpec
+    output_spec = ACMOutputSpec
+
+    def _run_interface(self, runtime):
+        allmaps = nb.concat_images(
+            [nb.load(f) for f in self.inputs.in_files]).get_data()
+        acm_pos = (allmaps > self.inputs.threshold).astype(
+            float).mean(3)
```

Comment:
What about:

```Python
acm_pos = np.mean(allmaps > self.inputs.threshold, axis=3, dtype=np.float32)
```

I think it reads more cleanly, and it'll make any line breaks cleaner looking, too. (I also prefer to use explicit `axis` and `dtype` keywords, just to avoid having to look up the order when I read. And I prefer to use actual dtypes, just to avoid depending on numpy magic that they can change at any time.)